terraform/variables.tf

```diff
@@ -23,6 +29,11 @@ variable "mongo_disks" {
   }
 }
 
+variable "mongo_instance_count" {
```

Comment:
This can probably also be given a default value of 1.

Reply:
Good call! Please see 46576b5 for that default being added.